### ISSUE TITLE
https://github.com/projecthorus/radiosonde_auto_rx/wiki/Airspy-Mini-Heatsink-Mods gone?

### ISSUE THREAD

**Issue by MikeTango** [projecthorus/radiosonde_auto_rx#969]
Hi,

i am trying to make heads and tails of possibly conflicting information. 

https://github.com/projecthorus/radiosonde_auto_rx/wiki/Airspy-Mini-vs-R2-Notes states that you can have performance issues when setting Airspy Mini to 20mhz sample size, with a higher noisefloor.

The config that i am running (1.8.0) has a warning that the radio overheats when doing so and needs additional heatsinking

# Uncomment this to run your Airspy Mini at 20 MHz sample rate (same at the Airspy R2)
# NOTE - This can result in your Airspy Mini overheating if you do not add additional thermal material inside it, and heatsink the case!
#samprate = 20000000

The corresponding page https://github.com/projecthorus/radiosonde_auto_rx/wiki/Airspy-Mini-Heatsink-Mods but is gone. 

I found this change from two weeks ago

https://github.com/projecthorus/radiosonde_auto_rx/wiki/KA9Q%E2%80%90Radio-Setup-Notes/_compare/af97464e1d1436f8d169eb3d4fcfdfabe01e0de8...828f7fa94a532cc43fe9b7cb6dc075f6e8bb6ee9

TL;DR is the heat problem a non issue now?

**Comment by darksidelemm** [comment]
It seems that heat isn't the issue, correct. Spurs are the issue (if they are an issue at all)

**Comment by MikeTango** [comment]
Would you mind adding the disassembly instructions again just for info? I am running the radio in an enclosed box and every bit counts.

**Comment by darksidelemm** [comment]
Doesn't look like I can revert the edits, sorry.... The best I can do is point you to these photos - https://photos.app.goo.gl/H815go2PG5B9LXm38

**Comment by MikeTango** [comment]
![IR_0743](https://github.com/user-attachments/assets/32f3e98b-ecb0-4506-bb64-fcf1aa4b91e9)
![IR_0744](https://github.com/user-attachments/assets/48d971aa-0ad7-4835-9383-fe47fcd058ae)

just took out the old trusty FLIR

**Comment by darksidelemm** [comment]
Sure...  but the case style makes it kind of difficult to just bond one small IC to the case. If you just use a small bit of thermal material, it's likely to get pulled off the board as you slide it into the case again.

**Comment by MikeTango** [comment]
I used some stuff that came with a raspi, there is a small copper piece that i put on top of the stacks, it slides in easy. The pressure on the big IC side pushes the board towards the other side of the casing and the chip on that side now has direct contact with the case. I admit that it's a bit tricky to slide in like that. For the external cooling fins i used some NVME2 format radiators that i had lying around. I tried active cooling but the fan motor plugged into the raspi GPIO introduced all kinds of interference. 
![signal-2025-01-01-113032](https://github.com/user-attachments/assets/d8b1b049-e742-4479-b5cb-86196fb4ede3)

This system is my portable, it's deployed in a marine environment and hasn't made any problems so far.

**Comment by MikeTango** [comment]
![Screenshot 2025-01-02 at 11-53-48 ❤️🧡💛💚💙💜 Dreamcatcher 💜💙💚💛🧡❤️ - Radiosonde Auto-RX Status](https://github.com/user-attachments/assets/025bfe9f-078b-47c6-bcce-2062ca4e6325)

This is from my permanent setup, no issues so far

**Comment by darksidelemm** [comment]
Oh the station with all the emojis that I get the occasional complaint about it YOU!

Keep it up :-)

Glad to hear it works!

**Comment by MikeTango** [comment]
It is absolutely not my fault that UTF-8 exists.

![sanitize](https://github.com/user-attachments/assets/1ced5f5a-2c11-4b31-a88b-231b2d5e7bbf)
